runtime/src/main/java/org/corfudb/util/Utils.java

```diff
@@ -232,6 +232,29 @@ public static long getLogTail(Layout layout, CorfuRuntime runtime) {
         return globalLogTail;
     }
 
+
+    public static long getMaxLogFileSize(Layout layout, CorfuRuntime runtime) {
```

Comment:
Using "File" is a little misleading because its more than one file, I would just use `LogSize`